wsmaster/che-core-api-metrics/pom.xml

```diff
@@ -59,6 +59,10 @@
             <groupId>org.eclipse.che.core</groupId>
             <artifactId>che-core-api-workspace-shared</artifactId>
         </dependency>
+        <dependency>
```

Comment:
Stuff in wsmaster shouldn't be tied to an infrastructure implementation. All the dependencies on kubernetes-specific stuff in wsmaster are currently only in the test scope. From code org pov, I don't think this is correct.